### ISSUE TITLE
CMS: Color contrast for buttons are not accessible on the event details page

### ISSUE THREAD

**Issue by lalov1** [utsa-asc/college-dls#807]
![Image](https://github.com/utsa-asc/college-dls/assets/7785251/c6dd295b-d4f5-4b2a-b990-545656b1767d)
![Image](https://github.com/utsa-asc/college-dls/assets/7785251/c05b50ee-9ce8-46f1-b558-b9f2e7d83d14)
